src/cli/peakdetector/peakdetectorcli.cpp

```diff
@@ -1135,22 +1119,6 @@ bool PeakDetectorCLI::_sendUserEmail(QMap<QString, QString> creds,
     return status;
 }
 
-void PeakDetectorCLI::saveMzRoll(string setName)
-{
-    if (saveMzrollFile == true) {
-#ifndef __APPLE__
-        double startSavingMzroll = getTime();
-#endif
-
-        writePeakTableXML(setName + ".mzroll");
```

Comment:
This function is not being used anywhere. Please remove it.

Reply:
Same for `writeSampleListXML`, `writeParametersXML` and `writeGroupXML`

Reply:
Sorry. Missed this. Removed now.